### PR TITLE
Fix: Offset search API pagination params

### PR DESCRIPTION
**Pull request by juliomoreira**
## What's the purpose of this pull request?

Fix paginating Search API for products static paths.

Related to: #726

### CODE REVIEW

File: packages/gatsby-source-vtex/src/staticPaths.ts [hunk 1]
```diff
@@ -100,8 +100,10 @@ const staticPaths = async ({
       fetchVTEX<Array<{ linkText?: string }>>(
         api.catalog.category.search({
           sort: 'OrderByTopSaleDESC',
-          from: pageIndex * itemsPerPage,
-          to: pageIndex * itemsPerPage + itemsPerPage,
+          from: pageIndex ? pageIndex * itemsPerPage + 1 : pageIndex,
```
Comment (tlgimenes) [inline packages/gatsby-source-vtex/src/staticPaths.ts:103]: isn't it easier to make something like:
```suggestion
          from: pageIndex * itemsPerPage,
          to: pageIndex * itemsPerPage + itemsPerPage - 1,
```

**Comment by juliomoreira** [comment]
@tlgimenes I wasn't able to test this in a local development environment yet, to further inspect the availability problem.

Whenever I try to use my local version either with `yarn link` or mapping it to `resolve` I get the following error:

```

 ERROR 

Cannot create as TypeComposer the following value:
  Query.



  Error: Cannot create as TypeComposer the following value: 
    Query.
  
  - SchemaComposer.js:463 SchemaComposer.createTempTC
    [storecomponents.store]/[graphql-compose]/lib/SchemaComposer.js:463:11
  
  - schema.js:763 forEach
    [storecomponents.store]/[gatsby]/src/schema/schema.js:763:36
  
  - Array.forEach
  
  - schema.js:761 addThirdPartySchemas
    [storecomponents.store]/[gatsby]/src/schema/schema.js:761:21
  
  - schema.js:209 updateSchemaComposer
    [storecomponents.store]/[gatsby]/src/schema/schema.js:209:9
  
  - schema.js:64 buildSchema
    [storecomponents.store]/[gatsby]/src/schema/schema.js:64:3
  
  - index.js:105 build
    [storecomponents.store]/[gatsby]/src/schema/index.js:105:18
  
  - build-schema.ts:19 buildSchema
    [storecomponents.store]/[gatsby]/src/services/build-schema.ts:19:3
  
  - index.ts:50 bootstrap
    [storecomponents.store]/[gatsby]/src/bootstrap/index.ts:50:3
  
  - build.ts:82 build
    [storecomponents.store]/[gatsby]/src/commands/build.ts:82:53
  

not finished building schema - 0.341s

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I've gone through all steps mentioned in [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/master/CONTRIBUTING.MD)

Is something missing from these instructions?

Thanks,

**Comment by juliomoreira** [comment]
Nevermind, I missed this section from README https://github.com/vtex/faststore#how-to-develop

**Comment by tlgimenes** [comment]
This was fixed on #738

**Pull request closed without merging**